source/Outfit.cpp

```diff
@@ -109,6 +109,7 @@ namespace {
 		{"discharge resistance energy", 0.},
 		{"discharge resistance fuel", 0.},
 		{"discharge resistance heat", 0.},
+		{"drag reduction", -0.99},
```

Comment:
I might be wrong, but with this you increase the drag by a 100fold?
1 -0.99 = 0.01
1 / 0.01 = 100

Reply:
but should this be the default?